### DISCUSSION TITLE
Can I use both Plug and Packer

### DISCUSSION THREAD

**Discussion by sriramkandukuri** [wbthomason/packer.nvim#629] [Q&A]
Hi,

I want to have both Plug and packer as below

Plug - For common vim and nvim plugins
Packer - For lua only nvim plugins

**Comment by nanotee** [comment]
Hi, you might want to take a look at #294 (the gist is that it's not recommended)

**Reply by wbthomason** [reply]
Slight addendum: if you want to use `packer` just for managing Luarocks, using both should be fine. Otherwise, while it may technically be possible to use both together, it's unsupported/at your own risk.
